IFC4x3/Sections/Resource definition data schemas/Schemas/IfcExternalReferenceResource/Entities/IfcDatasetReference/DocEntity.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcDatasetReference" UniqueId="b5e5d092-a5c0-4b1b-bde8-f9f8245ebe08" BaseDefinition="IfcExternalReference" EntityFlags="32">
+	<Attributes>
+		<DocAttribute Name="Description" UniqueId="ca30fa31-ac3e-4469-ae6f-26d92e6ff6b2" DefinedType="IfcText">
+			<Documentation>Description of the dataset reference for informational purposes.</Documentation>
+		</DocAttribute>
+		<DocAttribute Name="ReferencedDataset" UniqueId="b967d95b-2e3b-44f4-9251-a812c28ce43c" DefinedType="IfcDatasetInformation" />
+		<DocAttribute Name="Filter" UniqueId="3ec84a91-b491-4f35-ae65-fb5089d202df" DefinedType="IfcText" AttributeFlags="1">
+			<Documentation>An optional text as e.g. SQL, XQuery etc to filter out data when only a subset is relevant for the dataset reference.</Documentation>
+		</DocAttribute>
+		<DocAttribute Name="DatasetRefForObjects" UniqueId="81c4020c-6c5d-4142-984f-8fd28093be67" DefinedType="IfcRelAssociatesDataset" AggregationType="3" Inverse="RelatingDataset">
+			<Documentation>The dataset reference with which objects are associated.</Documentation>
+		</DocAttribute>
+	</Attributes>
+	<WhereRules>
+		<DocWhereRule Name="WR1" UniqueId="71089927-4b48-4fca-93b9-8a06d4e1ff88">
+			<Documentation>A name should only be given, if no dataset information (including the dataset name) is attached</Documentation>
```

Comment:
<Documentation>A name should only be given, if no dataset information (including the dataset name) is attached.</Documentation>

Reply:
Done (punctuation).